### DISCUSSION TITLE
Compile error on Unity 2021.3.29

### DISCUSSION THREAD

**Discussion by daneric** [wiverson/supabase-unity-template#3] [Q&A]
I opened the project in Unity 2021.3.29, the occurrence of the following errors came to pass:

Assembly 'Library/ScriptAssemblies/App.dll' will not be loaded due to errors:
Reference has errors 'Supabase.Gotrue'.

Assembly 'Assets/Supabase/gotrue-csharp.4.2.1/lib/netstandard2.0/Supabase.Gotrue.dll' will not be loaded due to errors:
Reference has errors 'JWTDecoder'.

Assembly 'Assets/Supabase/JWTDecoder.0.9.2/lib/netstandard2.0/JWTDecoder.dll' will not be loaded due to errors:
JWTDecoder references strong named Newtonsoft.Json Assembly references: 12.0.0.0 Found in project: 13.0.0.0.
Assembly Version Validation can be disabled in Player Settings "Assembly Version Validation"

Assembly 'Assets/Supabase/supabase-csharp.0.13.1/lib/netstandard2.0/Supabase.dll' will not be loaded due to errors:
Reference has errors 'Supabase.Gotrue'.

UPM packages: 

![image](https://github.com/wiverson/supabase-unity-template/assets/1892500/c2118638-8075-449a-9e6f-a35e498c31e0)

**Comment by wiverson** [comment]
Hmm, the template project was built for 2022 LTS and you're using 2021. You might be able to fix this by clicking on the .asmdef file in the App folder and updating the dependencies.
